src/test/compile-fail/issue-15524.rs

```diff
@@ -12,13 +12,16 @@ const N: isize = 1;
 
 enum Foo {
     A = 1,
-    B = 1, //~ ERROR discriminant value `1isize` already exists
-    //~^^ NOTE conflicting
+    B = 1, //~ ERROR discriminant value
+    //~^ NOTE enum already
+    //~^^^ NOTE first use
```

Comment:
Hi Jonathan!  I thought that line 17 was showing the `NOTE first use` exactly on line 14, I've tried modificating it but then it shows the note somewhere else.
`python2 src/bootstrap/bootstrap.py --step check-cfail issue-15524 --stage 1`
Is returning
`test [compile-fail] compile-fail/issue-15524.rs ... ok`